reporter/html.js

```diff
@@ -1,6 +1,14 @@
 import QUnit from "../src/core";
 import { window, navigator } from "../src/globals";
 
+const stats = {
+	totalTests: 0,
```

Comment:
Could we document the relationships between these values? It's worth commenting that `totalTests` equals the sum of the other values, and maybe even keeping two variables (e.g., `let totalTests = 0; const testPartitions = {…}`).